sap/5.7/modules/ROOT/pages/sap-connector-config-snc-kerberos.adoc

```diff
@@ -0,0 +1,149 @@
+= Configuring an SAP Secure Network Communication Connection with Kerberos
+
+Configure an SAP Secure Network Communication (SNC) with Kerberos to take advantage of security features such as secure data communication between the SAP system client and server, application-level end-to-end security, and the ability to change security products without impacting your SAP business applications. Enabling SNC with Kerberos requires you to configure the client environment, the SAP server, and Anypoint Connector for SAP (SAP Connector). 
+
+NOTE: SAP does not provide support for third-party solutions.
+
+== Before You Begin
+
+To configure an SAP SNC with Kerberos, you must have access to:
+
+* The SAP GUI application. To install this client, Java must already be installed on your machine. Then, download the installer for your operating system from SAP.
+* An SAP ECC instance that can use SNC system variables
+* The following transactions:
+** `STRUST` (Trust Manager)
+** `SM30` (Table Maintenance)
+** `SNC1` (Generate SNC name for user)
+** `SU01` (User Maintenance)
+** `SM59` (RFC Destinations)
+
+== Configure Active Directory
+
+Create a new Service Principal Name (SPN) according to the operating system you are using and export the keytab.
+
+If you are using Windows, follow these steps:
+
+. Create a service user `adminsap` in Active Directory.
+. Check *Password never expires* and uncheck *Use DES encryption types for this account*.
```

Comment:
Style Ref says to use select/deselect for checkboxes.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22select,%22%22choose%22